src/coreclr/tests/src/GC/API/GC/AddMemoryPressureTest.cs

```diff
@@ -132,51 +128,40 @@ public bool TooLargeTest()
         return retVal;
     }
 
-
     public bool StressTest()
     {
         TestCount++;
 
         Console.WriteLine("StressTest Started...");
 
         int gcCount1 = createDummies(true);
-
-
         int gcCount2 = createDummies(false);
 
         Console.WriteLine("{0} {1}", gcCount1, gcCount2);
         if (gcCount1 > gcCount2)
         {
-            Console.WriteLine("StressTest Passed");
-            Console.WriteLine();
+            Console.WriteLine("StressTest Passed\n");
             return true;
         }
 
-        Console.WriteLine("StressTest Failed");
-
-        Console.WriteLine();
+        Console.WriteLine($"StressTest Failed: {gcCount1} {gcCount2}\n");
         return false;
     }
 
-
     private int createDummies(bool heavy)
     {
         int gcCount = GC.CollectionCount(0);
 
         for (int i = 0; i < 100; i++)
         {
-            Dummy dummy = new Dummy(heavy);
-            int gen = GC.GetGeneration(dummy);
-            if (gen != 0)
-            {
-                Console.WriteLine("Warning: newly-allocated dummy ended up in gen {0}", gen);
-            }
+            AddMemoryPressureDummy dummy = new AddMemoryPressureDummy(heavy);
+            GC.GetGeneration(dummy);
+            AddMemoryPressureDummy.RemovePressure(heavy);
```

Comment:
You probably want to do Remove pressure after adding all the pressure and checking for GCs. It might make the test more predictable, but not 100% predictable. 

I am not sure why allocating dummy objects is needed at all. This scenario is strange.

Reply:
I am removing the pressure with `AddMemoryPressureDummy.RemovePressure(heavy);`. The addition is done in `AddMemoryPressureDummy`'s constructor.

>I am not sure why allocating dummy objects is needed at all. This scenario is strange.

I agree. I am inclined to remove this stress test entirely and file an issue for adding an actual reliable test for this scenario. Any objections?

Reply:
Actually the libraries has tests for the API boundary issues. 

https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/tests/System/GCTests.cs#L21-L30

https://github.com/dotnet/runtime/blob/4eaccd61b1a2a4b078140f8b36f2b45ab5980e8e/src/libraries/System.Runtime/tests/System/GCTests.cs#L334-L343

I will add an issue and remove these tests entirely.

Reply:
No objections from me. If we have stress issues here, we will not learn about that form this test.